.github/workflows/index-monitor.yml

```
@@ -0,0 +1,120 @@
+name: '[Index] Monitor remote index.yaml'
+
+on:
+  schedule:
+    # Every 10 minutes
+    - cron: '*/10 * * * *'
+
+# Remove all permissions by default
+permissions: {}
+
+jobs:
+  integrity-check:
+    name: Compare the index.yaml checksums remote and locally
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    outputs:
+      result: ${{ steps.integrity-check.outputs.result }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          ref: 'index'
+      - name: Check index integrity
+        id: integrity-check
+        run: |
+          status="fail"
+          attempts=0
+          # We want to check for consistent failures
+          # To do so, we will look for 3 consecutive failures with a 30 seconds wait
+          # A single success is enough to pass
+          while [[ "${status}" != "ok" && $attempts -lt 3 ]]; do
+            # Check the index.yaml integrity
+            REMOTE_MD5=($(curl -Ls https://charts.bitnami.com/bitnami/index.yaml | md5sum))
+            REPOSITORY_MD5=($(md5sum bitnami/index.yaml))
+            # Compare the index.yaml checksums remote and locally
+            if [[ "${REPOSITORY_MD5[0]}" == "${REMOTE_MD5[0]}" ]]; then
+              status='ok'
+            else
+              attempts=$((attempts+1))
+              echo "Integrity check failed. Remote checksum '${REMOTE_MD5[0]}' does not match expected '${REPOSITORY_MD5[0]}'";
+              # Refresh the 'index' branch in case it was updated
+              git fetch origin index
+              git reset --hard origin/index
+              # Wait 30 seconds
+              sleep 30
+            fi
+          done
+          echo "result=${status}" >> $GITHUB_OUTPUT
+      - name: Show messages
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea
+        with:
+          script: |
+            if ("${{ steps.integrity-check.outputs.result }}" != "ok" ) {
+              core.setFailed("Integrity check failed");
+            } else {
+              core.info("Integrity check succeeded")
+            }
```

I think we could simplify this piece of code with something like this:
```suggestion
          if [[ "${status}" != "ok" ]]; then
            echo "::error:: Integrity check failed"
          else
            echo "::info:: Integrity check succeeded"
          fi
          echo "result=${status}" >> $GITHUB_OUTPUT
```